homeassistant/components/homekit_controller/connection.py

```diff
@@ -118,6 +122,17 @@ def remove_pollable_characteristics(self, accessory_id):
             char for char in self.pollable_characteristics if char[0] != accessory_id
         ]
 
+    def add_watchable_characteristics(self, characteristics):
+        """Add (aid, iid) pairs that we need to poll."""
+        self.watchable_characteristics.extend(characteristics)
+        self.hass.add_job(self.pairing.subscribe(characteristics))
```

Comment:
If the target is a coroutine, we should use `hass.async_create_task` instead of `hass.async_add_job`. The latter is legacy for general use.